addons/base_import_module/static/src/base_import_list_renderer.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module */
+import { useService } from "@web/core/utils/hooks";
+import { ListRenderer } from "@web/views/list/list_renderer";
+
+export class ImportListRenderer extends ListRenderer {
+
+    setup() {
+        super.setup()
+        this.action = useService("action");
+    }
+
+    get hasSelectors() {
+        return super.hasSelectors && this.props.list.records[0]._values.module_type != 'industries';
```

Comment:
This raises an error when you have no records (like if you search with a typo, no result, OwlError
```suggestion
        return super.hasSelectors && this.props.list.records[0]?._values.module_type != 'industries';
```